modules/requesting-a-shap-explanation-using-cli.adoc

```diff
@@ -0,0 +1,144 @@
+----
+
+. Set an environment variable to define the name of your model.
++
+----
+export MODEL="model-name"
+----
+
+. Use `GET /info/inference/ids/${MODEL}` to get a list of all inference IDs within your model inference data set.
++
+[source]
+----
+curl -skv -H "Authorization: Bearer ${TOKEN}" \
+   https://${TRUSTY_ROUTE}/info/inference/ids/${MODEL}?type=organic
+----
++
+You see output similar to the following:
++
+[source]
+----
+[
+  {
+    "id":"a3d3d4a2-93f6-4a23-aedb-051416ecf84f",
+    "timestamp":"2024-06-25T09:06:28.75701201"
+  }
+]
+----
+
+. Set environment variables to define the two latest inference IDs (highest and lowest predictions).
++
+[source]
+----
+export ID_LOWEST=$(curl -s ${TRUSTY_ROUTE}/info/inference/ids/${MODEL}?type=organic | jq -r '.[-1].id')
+
+export ID_HIGHEST=$(curl -s ${TRUSTY_ROUTE}/info/inference/ids/${MODEL}?type=organic | jq -r '.[-2].id')
+----
+
+. Use `POST /explainers/local/shap` to request the SHAP explanation with the following syntax and payload structure:
++
+*Sytnax*:
++
+----
+curl -sk -H "Authorization: Bearer ${TOKEN}" -X POST \
+ -H "Content-Type: application/json" \
+ -d <payload>
+----
++
+*Payload structure*:
+
+`PredictionId`:: The inference ID.
+`config`:: The configuration for the SHAP explanation, including `model` and `explainer` parameters. For more information, see link:https://trustyai-explainability.github.io/trustyai-site/main/trustyai-service-api-reference.html#ModelConfig[Model configuration parameters] and link:https://trustyai-explainability.github.io/trustyai-site/main/trustyai-service-api-reference.html#SHAPExplainerConfig[SHAP explainer configuration parameters].
+
+For example:
+
+[source]
+----
+echo "Requesting SHAP for lowest"
+curl -sk -H "Authorization: Bearer ${TOKEN}" -X POST \
+  -H "Content-Type: application/json" \
+  -d "{
+    \"predictionId\": \"$ID_LOWEST\",
+    \"config\": {
+        \"model\": { <1>
+            \"target\": \"modelmesh-serving:8033\", <2>
+            \"name\": \"${MODEL}\",
+            \"version\": \"v1\"
+        },
+        \"explainer\": { <3>
+          \"n_samples\": 75
+        }
+    }
+  }" \
+  ${TRUSTYAI_ROUTE}/explainers/local/shap
+----
+
+[source]
+----
+echo "Requesting SHAP for highest"
+curl -sk -H "Authorization: Bearer ${TOKEN}" -X POST \
+    -H "Content-Type: application/json" \
+    -d "{
+        \"predictionId\": \"$ID_HIGHEST\",
+        \"config\": {
+            \"model\": { <1>
+                \"target\": \"modelmesh-serving:8033\", <2>
+                \"name\": \"${MODEL}\",
+                \"version\": \"v1\"
+            },
+            \"explainer\": { <3>
+              \"n_samples\": 75
+            }
+        }
+    }" \
+    ${TRUSTYAI_ROUTE}/explainers/local/shap
+
+----
+<1> Specifies configuration for the model. For options, see link:https://trustyai-explainability.github.io/trustyai-site/main/trustyai-service-api-reference.html#ModelConfig[Model configuration parameters].
```

Comment:
Regarding, "For options," could this be made a little clearer? 

"For more information about the model configuration options, see..."